build_tools/bazel/third_party_import/llvm-project/overlay/mlir/test/BUILD.bazel

```diff
@@ -2,7 +2,7 @@ load("@org_tensorflow//third_party/mlir:tblgen.bzl", "gentbl")
 
 licenses(["notice"])
 
-package(default_visibility = ["//visibility:public"])
+package(default_visibility = [":test_friends"])
```

Comment:
I swear this change happens every time, which makes me think that copybara is doing some stupid rewrite stuff...

Reply:
Yeah copybara is rewriting these on import. Lemme see about fixing that.